backend/node_engine/email_nodes.py

```
@@ -1,678 +1,588 @@
+        # Normalize recipients
+        if "recipients" in cleaned and isinstance(cleaned["recipients"], list):
+            cleaned["recipients"] = [r.strip().lower() for r in cleaned["recipients"]]
+
+        # Ensure date is properly formatted
+        if "date" in cleaned:
+            try:
+                # Try to parse and reformat date
+                if isinstance(cleaned["date"], str):
+                    # Assume ISO format, could add more parsing
+                    datetime.fromisoformat(cleaned["date"].replace('Z', '+00:00'))
+            except:
```

Ruff (0.14.2)

Do not use bare `except`

(E722)